main/views.py

```diff
@@ -222,8 +219,10 @@ def fetch_fitbit_data(fitbit_member, access_token):
                                                             end_date=arrow.utcnow().format('YYYY-MM-DD'))
         print("Fetching data from: ", final_url)
         # Fetch the data
-        r = rr.get(url=final_url, headers=headers, realms=["Fitbit"])
+        r = make_request_respectful_get.delay('final_url', realms=["Fitbit"], headers=headers)
         # Append the results to results dictionary with url "name" as the key
+        print("printing r:")
+        print(r)
```

Comment:
Probably want to eventually take the print statements out

Reply:
Absolutely- it still isn't working (`r` is now an async object and I need to be able to access the data but haven't worked out how to do that) so keeping them there until it runs :)